t/t0021-remote-powerman.t

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+
+test_description='Check powermand as a power control device'
+
+. `dirname $0`/sharness.sh
+
+powermand=$SHARNESS_BUILD_DIRECTORY/src/powerman/powermand
+powerman=$SHARNESS_BUILD_DIRECTORY/src/powerman/powerman
+vpcd=$SHARNESS_BUILD_DIRECTORY/test/vpcd
+vpcdev=$SHARNESS_TEST_SRCDIR/../etc/devices/vpc.dev
+powermandev=$SHARNESS_TEST_SRCDIR/../etc/devices/powerman.dev
+
+# Use port = 11000 + test number
+# That way there won't be port conflicts with make -j
+testaddr=localhost:11021
+
+
+makeoutput() {
+	printf "on:      %s\n" $1
+	printf "off:     %s\n" $2
+	printf "unknown: %s\n" $3
+}
+
+test_expect_success 'create controlled powerman.conf with 4 devices (64 plugs)' '
+	cat >rpowerman.conf <<-EOT
+	include "$vpcdev"
+	device "test0" "vpc" "$vpcd |&"
+	device "test1" "vpc" "$vpcd |&"
+	device "test2" "vpc" "$vpcd |&"
+	device "test3" "vpc" "$vpcd |&"
+	node "t[0-15]"  "test0"
+	node "t[16-31]" "test1"
+	node "t[32-47]" "test2"
+	node "t[48-63]" "test3"
+	EOT
+'
+test_expect_success 'create controlling powerman.conf' '
+	cat >powerman.conf <<-EOT
+	listen "$testaddr"
+	include "$powermandev"
+	device "p0" "powerman" "$powermand --stdio -c rpowerman.conf -f |&"
```

Comment:
Not necessarily for this PR but --stdio not clear of purpose.  Not in manpage.

Reply:
Opened #110